docs/source/exceptions.rst

```diff
@@ -1,11 +1,20 @@
+
+
+===============
 Exceptions
-=============================
-This package contains all Snowpark client-side exceptions.
```

Comment:
These comments are auto extracted by `automodule` directive, I used `noindex` to avoid duplicate indexing of the methods/classes.